core/src/components/select-popover/test/basic/select-popover.e2e.ts

```diff
@@ -66,3 +66,19 @@ configs({ modes: ['md'], directions: ['ltr'] }).forEach(({ title, config }) => {
     });
   });
 });
+
+/**
+ * This behavior does not vary across directions.
+ * The components used inside of `ion-select-popover`
+ * do have RTL logic, but those are tested in their
+ * respective component test files.
+ */
+configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('select-popover: rendering'), () => {
```

Comment:
Could we also have one for a select with multiple selection enabled, since the styling is different there? (I know I'm going to want one with my upcoming change to use the new form control syntax, at least.)

Reply:
Added https://github.com/ionic-team/ionic-framework/pull/27393/commits/09d71de8194cca51feac45225c91e90a5053d51f